docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -27,7 +27,15 @@ ifndef::no-reactive-routes[]
 endif::no-reactive-routes[]
 .
 As a result, applications that use the TLS Registry can be ready to handle secure communications out of the box.
-TLS Registry also provides features like automatic certificate reloading, Let's Encrypt (ACME) integration, Kubernetes Cert-Manager support, and compatibility with various keystore formats, such as PKCS12, PEM, and JKS.
+
+TLS Registry also provides automatic certificate reloading
+ifndef::no-lets-encrypt[]
+, integration with Let's Encrypt (ACME)
```

Comment:
This is not needed since the `,` you are suggesting here is present in the next line:
![image](https://github.com/user-attachments/assets/a216b591-0914-458a-8584-40a4b35c274b)

Reply:
I see, now. +1